### PR TITLE
[#122] Update Slot Size to 10 Minutes

### LINKED ISSUE #109: Fix reservation date picker to only show future dates

### LINKED ISSUE #122: Update Slot Size to 10 Minutes
According to our requirements: "Terminbuchungsintervall = Buchbarer Ticket beginn alle 10 Minuten"

### PR DESCRIPTION
**Pull request by juliaringler**
* Updated slot size to 10 minutes
* Updated startTime for getLocationReservations
* Wrapped ReserveSlot button with Center for a nicer animation
* Fixed bug with date picker allowing yesterday to be selected

### CODE REVIEW

File: lib/ui/map/location_detail_sheet.dart [hunk 1]
```diff
@@ -175,8 +184,8 @@ class _ChangeDateButton extends StatelessWidget {
         Future<DateTime> selectedDate = showDatePicker(
           context: context,
           initialDate: this.selectedDate,
-          firstDate: DateTime.now().subtract(Duration(days: 1)),
```
Comment (juliaringler) [inline lib/ui/map/location_detail_sheet.dart:172]: I was not able to reproduce your bug @Johannes11833 but I strongly suspect that this part caused the issue.

File: lib/ui/map/location_detail_sheet.dart [hunk 2]
```diff
@@ -248,40 +270,37 @@ class _LocationInformation extends StatelessWidget {
         Padding(
           padding: const EdgeInsets.all(8.0),
           child: Container(
-            height: 75,
```
Comment (juliaringler) [inline lib/ui/map/location_detail_sheet.dart:245]: this part here improves the existing layout.
Old:
old1
old2

New:
new1
new2

**Review by ConstantinB9 — DISMISSED** [review]

File: lib/ui/map/location_detail_sheet.dart (hunk 2 above)
Comment (ConstantinB9) [reply lib/ui/map/location_detail_sheet.dart:245]: you have swapped old and new, right?

**Review by ConstantinB9 — APPROVED** [review]

File: lib/ui/map/location_detail_sheet.dart (hunk 2 above)
Comment (juliaringler) [reply lib/ui/map/location_detail_sheet.dart:245]: No …

**Pull request merged**